net/syncthing-0.14/Portfile

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           obsolete 1.0
```

Comment:
Optional, but I like to add a note in comment about when this was removed (when it can be removed), so that if anyone wants to do a cleanup after a while, it's easy to figure out whether a year has passed or not (we would need an automated way for that, we just don't have anything in place yet).